Assets/MRTK/SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -444,15 +447,19 @@ protected virtual void AttachToNewTrackedObject()
 
         private void TrackTransform(Transform target)
         {
-            if (trackingTarget != null || target == null) return;
+            if (trackingTarget == null)
+            {
+                trackingTarget = new GameObject("SolverHandler Tracking Target");
+                trackingTarget.hideFlags = HideFlags.HideInHierarchy;
+            }
 
-            string name = string.Format("SolverHandler Target on {0} with offset {1}, {2}", target.gameObject.name, AdditionalOffset, AdditionalRotation);
```

Comment:
It seems unfortunate to lose all this additional information in the gameobject name :/ if there's no actual value in it, then I guess it's fine, but I'm sure it's helpful in the editor at least